1/styles/scss/_media-mobile.scss

```diff
@@ -0,0 +1,158 @@
+@media screen and (max-width: 576px) {
+  .container{
+    max-width: 80vw;
+  }
+  .heading h1{
+    font-size: 40px;
+    padding: 140px 0;
+  }
+  .header-menu .active::after{
+    left: 45%;
+  }
+  .logo{
+    padding: 20px 30px;
+  }
+  .local-nav {
+    z-index: 0;
+    left: 13%;
+    li {
+      padding: 10px 25px;
+    }
+  }
+  .tips{
+    .header{
+      margin-top: 30px;
+      svg{
+        width: 160px;
+      }
+    }
+  }
+  .accordion{
+    margin-top: 20px;
+
+    .number{
+      padding: 0 10px;
+      position: relative;
+      top: 10px;
+    }
+    .accordion-item-header{
+      padding: 0 0 20px 0;
+      font-size: 16px;
+      align-items: flex-end;
+    }
+    .plus{
+      padding: 0;
+    }
+  }
+  .accordion-item.show{
+    .accordion-item-header{
+      font-size: 24px;
+    }
+    .accordion-item-content{
+      flex-direction: column;
+    }
+    .img-wrapper{
+      max-width: 160px;
+      margin: 0 auto;
+      img{
+        width: 100%;
+      }
+    }
+    .text-wrapper{
+      font-size: 14px;
+
+    }
+  }
+  .glossary{
+    margin-top: 40px;
```

Comment:
doesn't match the design
![image](https://user-images.githubusercontent.com/58749210/119579089-61633d00-bdc6-11eb-8491-fe72b33f274d.png)

Reply:
done